astropy/modeling/tests/test_mappings.py

```diff
@@ -57,8 +66,21 @@ def test_identity():
     assert_allclose(model(1, 2), (-2.098076211353316, 2.3660254037844393))
     res_x, res_y = model(x, y)
     assert_allclose((res_x, res_y),
-                       (np.array([[-1.73205081, -1.73205081, -1.73205081],
-                                  [-1.73205081, -1.73205081, -1.73205081]]),
-                        np.array([[ 1.,  1.,  1.],
-                                  [ 1.,  1.,  1.]])))
+                    (np.array([[-1.73205081, -1.73205081, -1.73205081],
+                               [-1.73205081, -1.73205081, -1.73205081]]),
+                     np.array([[1., 1., 1.],
+                               [1., 1., 1.]])))
     assert_allclose(model.inverse(res_x, res_y), (x, y), atol=1.e-10)
+
+
+@pytest.mark.skipif('not HAS_SCIPY')
+def test_fittable_compound():
```

Comment:
@pllim Could you add the issue or PR number in a docstring. I have the feeling this test won't make much sense to me in a few months.